dora/core/server/master/src/main/java/alluxio/master/scheduler/Scheduler.java

```diff
@@ -461,8 +555,7 @@ public boolean enqueueTaskForWorker(@Nullable WorkerInfo workerInfo, Task task,
             LOG.error("Unexpected exception thrown in response future listener.", e);
             job.failJob(new InternalRuntimeException(e));
           } finally {
-            // whether task succeed or fail, remove it from q,
-            workerTaskQ.remove(task);
+            tasksQ.remove(task);
             mJobToRunningTasks.compute(job, (k, v) -> {
```

Comment:
feel like mJobToRunningTasks could be replaced by taskQ but we can do it in a later PR

Reply:
no no, mJobToRunningTasks indicates job to its list of task map, taskq is for tracking worker to tasks. we need both